libbeat/common/field.go

```diff
@@ -39,17 +39,22 @@ type Field struct {
 	Searchable   *bool  `config:"searchable"`
 	Aggregatable *bool  `config:"aggregatable"`
 	Script       string `config:"script"`
-	// Kibana field format specific
-	Pattern         string `config:"pattern"`
-	InputFormat     string `config:"input_format"`
-	OutputFormat    string `config:"output_format"`
-	OutputPrecision *int   `config:"output_precision"`
-	LabelTemplate   string `config:"label_template"`
-	UrlTemplate     string `config:"url_template"`
+	//Kibana params
```

Comment:
You didn't like the space? :-D

Reply:
need to change my editor config for comments ;)